packages/daemon/src/model/plugin.ts

```diff
@@ -49,24 +49,22 @@ export class PluginModel extends Model {
     return (await PluginModel.findAll({ where })).map(plugin => plugin.toJSON() as PluginEntity);
   }
 
-  static async query(filter?: ListPluginsFilter): Promise<PluginEntity[]> {
-    const where = {} as any;
-    if (filter.category) {
-      where.category = filter.category;
-    }
-    if (filter.datatype) {
-      where.datatype = filter.datatype;
-    }
-    return (await PluginModel.findAll({ where })).map(plugin => plugin.toJSON() as PluginEntity);
-  }
-
   static async findById(id: string): Promise<PluginEntity> {
     return (await PluginModel.findOne({ where: { id } }))?.toJSON() as PluginEntity;
   }
 
   static async findByIds(ids: string[]): Promise<PluginEntity[]> {
     return (await PluginModel.findAll({ where: { id: ids } })).map(plugin => plugin.toJSON() as PluginEntity);
   }
+
+  static async findByPrefixId(prefixId: string): Promise<PluginEntity[]> {
```

Comment:
```suggestion
  static async searchById(keyword: string): Promise<PluginEntity[]> {
```

Reply:
It's a prefix of id, not keyword.

Reply:
I don't think we should introduce the prefix of id in daemon, which would add the complexity of our models.